.github/CODEOWNERS

```
@@ -112,6 +112,7 @@ build-utils/        @getsentry/owners-js-build
 
 ### Visibility ###
 /src/sentry/api/endpoints/organization_tags.py                  @getsentry/visibility
+/src/sentry/api/endpoints/organization_events_histogram.py			@getsentry/visibility
```

```suggestion
/src/sentry/api/endpoints/organization_events_histogram.py      @getsentry/visibility
```

looks like the rest of these use spaces